partner/partner-management-service/src/main/java/io/mosip/pms/exception/PartnerServiceResponseExceptionHandler.java

```diff
@@ -48,25 +46,24 @@ public class PartnerServiceResponseExceptionHandler extends ResponseEntityExcept
 	String msg = "mosip.partnermanagement";
 	String version = "1.0";
 
-	@Override
-	protected ResponseEntity<Object> handleMethodArgumentNotValid(MethodArgumentNotValidException ex,
-			HttpHeaders headers, HttpStatus status, WebRequest request) {
+	public ResponseEntity<Object> handleMethodArgumentNotValid(MethodArgumentNotValidException ex) {
```

Comment:
why override is removed

Reply:
handleMethodArgumentNotValid Method does not override the method from its superclass  ResponseEntityExceptionHandler  class

Reply:
updated